crates/core/src/post/service.rs

```diff
@@ -55,4 +61,22 @@ impl PostService {
 
         Ok(post)
     }
+
+    pub async fn create_comment(&self, dto: CreateCommentDto) -> Result<Post> {
+        let record = self
+            .repository
+            .insert(InsertPostDto {
+                id: Post::generate_id()?.to_string(),
+                author_id: dto.author_id,
+                parent_id: dto.parent_id,
+                head: dto.parent_id.is_none(),
+                title: "".to_string(),
```

Comment:
In the future we are likely to have a `Option<String>` for this

Reply:
Ok! 🚀